.travis.yml

```diff
@@ -7,8 +7,8 @@ jobs:
   include:
 
     - stage: Linux - Conan develop
-      name: Python 3.5
-      python: 3.5
+      name: Python 3.6
```

Comment:
`subprocess.run` is Python 3.5, but it doesn't work here, probably the patch version is incompatible. https://docs.python.org/3.5/library/subprocess.html#subprocess.run